arbnode/batch_poster.go

```diff
@@ -235,6 +235,7 @@ func (b *BatchPoster) checkReverts(ctx context.Context, from, to *big.Int) (bool
 					return false, fmt.Errorf("getting a receipt for transaction: %v, %w", tx.Hash(), err)
 				}
 				if r.Status == types.ReceiptStatusFailed {
+					log.Error("Transaction: %v from batch poster was reverted", tx.Hash())
```

Comment:
The logging framework we (and geth) use takes its arguments in a bit of a weird way. Instead of format strings, it takes in a message and then a list of key-value arguments. E.g. `log.Info("something went wrong", "block", 123, "error", err)`. Currently this'd give a weird log line error complaining about an even number of arguments. To fix that we can use `log.Error("Transaction from batch poster reverted", "nonce", tx.Nonce(), "txHash", tx.Hash(), "blockNumber", r.BlockNumber, "blockHash", r.BlockHash)` (I added a few other fields that might be useful).

Reply:
Thanks! I keep forgetting that.